examples/metric/mae.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+def ae(y_true, y_pred):
```

Comment:
Maybe rename this to mae (Mean Absolute Error)

Reply:
sure. I added the .mean() and updated the filename but for some reason forgot to update function name